service/src/test/java/greencity/service/SuperAdminServiceImplTest.java

```diff
@@ -219,6 +219,89 @@ void addService() {
         verify(modelMapper).map(service, CreateServiceDto.class);
     }
 
+    @Test
+    void getAllCouriers() {
+        when(courierLocationRepository.findAllInfoAboutCourier()).thenReturn(List.of(ModelUtils.getCourierLocations()));
+        when(modelMapper.map(ModelUtils.getCourierLocations(), GetCourierLocationDto.class))
+            .thenReturn(ModelUtils.getCourierLocationsDto());
+
+        assertEquals(List.of(ModelUtils.getCourierLocationsDto()), superAdminService.getAllCouriersAndLocations());
+    }
+
+    @Test
+    void setCourierLimitBySumOfOrder() {
+        EditPriceOfOrder dto = ModelUtils.getEditPriceOfOrder();
+        CourierLocation courierLocation = ModelUtils.getCourierLocations();
+        when(courierLocationRepository.findCourierLocationsLimitsByCourierIdAndLocationId(1L, dto.getLocationId()))
+            .thenReturn(courierLocation);
+        when(courierLocationRepository.save(courierLocation)).thenReturn(courierLocation);
+
+        superAdminService.setCourierLimitBySumOfOrder(1L, dto);
+
+        verify(courierLocationRepository).findCourierLocationsLimitsByCourierIdAndLocationId(1L, dto.getLocationId());
+        verify(courierLocationRepository).save(courierLocation);
+    }
+
+    @Test
+    void setCourierLimitByAmountOfBag() {
+        EditAmountOfBagDto dto = ModelUtils.getAmountOfBagDto();
+        CourierLocation courierLocation = ModelUtils.getCourierLocations();
+
+        when(courierLocationRepository.findCourierLocationsLimitsByCourierIdAndLocationId(1L, dto.getLocationId()))
+            .thenReturn(courierLocation);
+        when(courierLocationRepository.save(courierLocation)).thenReturn(courierLocation);
+        superAdminService.setCourierLimitByAmountOfBag(1L, dto);
+
+        verify(courierLocationRepository).findCourierLocationsLimitsByCourierIdAndLocationId(1L, dto.getLocationId());
+        verify(courierLocationRepository).save(courierLocation);
+    }
+
+    @Test
+    void setLimitDescription() {
+        Courier courier = ModelUtils.getCourier();
+        CourierTranslation courierTranslation = CourierTranslation.builder()
+                .id(1L)
+                .name("Test")
+                .language(getLanguage())
+                .courier(courier)
+                .build();
+        CourierLocation courierLocation = ModelUtils.getCourierLocations();
+
+        lenient().when(courierRepository.findById(1L)).thenReturn(Optional.of(courier));
+        lenient().when(courierTranslationRepository.findCourierTranslationByCourier(courierLocation.getCourier())).thenReturn(courierTranslation);
+        lenient().when(courierTranslationRepository.save(courierTranslation)).thenReturn(courierTranslation);
```

Comment:
Remove lenient(). If you've got the UnnecessaryStubbingException remove this 3 method's;